### DISCUSSION TITLE
Feature: multiple choice field - limit maximum choices to make for a user

### DISCUSSION THREAD

**Discussion by Marco-Rinia** [umbraco/Umbraco.Forms.Issues#1511] [Ideas]
In some use cases it would be very usefull to **limit the number of maximum choices** for a user to make in a multiple choice field.

So maybe could this field be expanded with this functionality.

Regards, 

Marco

**Comment by TvGessel** [comment]
Thank you for the suggestion. We have added this to our internal backlog for further discussion and prioritization to determine if and when we can include this in the roadmap.
